### ISSUE TITLE
Error: Attribute "style" has already been defined

### ISSUE THREAD

**Issue by ryandzhunter** [uber/rides-android-sdk#1]
I have compile uber sdk on my project, but I found this issue

"Attribute "style" has already been defined" in color.xml. I dont have anything attribute named "style" on my own color.xml. So, what should I do to fix this issue?

Thank you

**Comment by mnemonicflow** [comment]
This error is due a conflict with other another library you are using in your project. 

The problem is that the same attribute name is used more than once, which is not allowed by the Android SDK build tools, which is due to bad design: https://code.google.com/p/android/issues/detail?id=22576

Android Studio locating the error in colors.xml is most certainly a bug in the IDE or the tools it runs that tell it the error.

Uber rides-android-sdk defines a custom attribute with the name 'style' for the RequestButton custom view at https://github.com/uber/rides-android-sdk/blob/master/sdk/src/main/res/values/attrs.xml#L29 

```xml

<resources>
    <attr name="uberButtonStyle" format="reference"/>

    <declare-styleable name="RequestButton">
        <attr name="client_id" format="string"/>
        <attr name="style" format="enum">
            <enum name="black" value="0"/>
            <enum name="white" value="1"/>
        </attr>
    </declare-styleable>
</resources>
```

@ryandzhunter
To find which is in conflict with rides-android-sdk: either exclude each library one by one or search for name="style" inside the sources of those libraries to determine which one has a declare-styleable with an attribute name of "style"

@itstexter 
As recommended by an official Android developer on the above issue from Google code, **to actually fix this problem rides-android-sdk needs to prefix the attribute names** used in custom views so that the name of each attribute is unique across all libraries one might use in a project, besides the already provided XML namespace.

`We recommend that you put a prefix to all library resources so that conflicts can be avoided.`

**Comment by itstexter** [comment]
@mnemonicflow is totally right here, I should rename the style attribute to something like "com_uber_style" will look to do this in the next release

**Comment by itstexter** [comment]
Sorry for the delay on this! Fixed in release [v0.3.0](https://github.com/uber/rides-android-sdk/releases/tag/v0.3.0)
